### PR TITLE
Regular cleanup of deprecated APIs

### PR DESCRIPTION
**Pull request by turansky**
cc @aerialist7 

Previous - #662

### CODE REVIEW

File: kotlin-react/src/main/kotlin/react/RBuilder.kt [hunk 1]
```diff
@@ -133,22 +106,6 @@ open class RBuilderImpl : RBuilder {
 
 open class RBuilderMultiple : RBuilderImpl()
 
-@Deprecated(message = "Unsafe API, use createElement")
-fun <T : RBuilder> buildElements(builder: T, handler: T.() -> Unit): dynamic {
```
Comment (Leonya) [inline kotlin-react/src/main/kotlin/react/RBuilder.kt:137]: Still using this one.

**Comment by Leonya** [comment]
renderEach can actually be removed.

File: kotlin-react/src/main/kotlin/react/RBuilder.kt (hunk 1 above)
Comment (turansky) [reply kotlin-react/src/main/kotlin/react/RBuilder.kt]: Restored

**Comment by turansky** [comment]
`renderEach` removed

**Pull request merged**